libs/components/src/next/form/datepicker/vl-datepicker.uig-css.ts

```diff
@@ -0,0 +1,40 @@
+import { css, CSSResult } from 'lit';
+
+const styles: CSSResult = css`
+    :host {
+        --vl-error-color: rgb(210, 55, 60);
+        --vl-success-color: rgb(0, 158, 71);
+    }
+    button {
+        cursor: pointer;
+    }
+    .vl-input-field {
+        border-radius: 0.3rem 0px 0px 0.3rem;
+        border-right-width: 0px;
+    }
+    .vl-input-addon--success {
+        border-color: var(--vl-success-color);
+    }
+    .vl-input-addon--success .vl-vi {
+        color: var(--vl-success-color) !important;
+    }
+    .vl-input-addon--error {
+        border-color: var(--vl-error-color);
+    }
+    .vl-input-addon--error .vl-vi {
+        color: var(--vl-error-color) !important;
+    }
+
+    .flatpickr-calendar {
+        z-index: 1000 !important;
+    }
+
+    .flatpickr-wrapper--block {
+        width: 100%;
+    }
+
+    .flatpickr-calendar--push-top {
+        margin-top: 14px;
```

Comment:
Rem waarden gebruiken aub

Reply:
Deze klasse is een overblijfsel van de originele variant waarbij die `visual-format` ook nog included was (en die zorgde voor een veel complexer component waar dan ook een heleboel spuug-en plakwerk voor nodig was, .o.a. dynamisch zetten van deze klasse). Deze is niet meer nodig - verwijderd.